genFigures.py

```diff
@@ -86,3 +86,10 @@
         # Overlay Figure 3b cartoon
         overlayCartoon(fdir + 'figure3B.svg',
                        f'{cartoon_dir}/figure3B.svg', 50, 40, scalee=0.24)
+
+    if sys.argv[1] == '0':
+        # Overlay Figure 0 cartoon
+        overlayCartoon(fdir + 'figure0.svg',
+                       f'{cartoon_dir}/lineage_uncensored.svg', 0, 0, scalee=0.7)
+        overlayCartoon(fdir + 'figure0.svg',
+                       f'{cartoon_dir}/lineage_censored.svg', 200, 0, scalee=0.7)
```

Comment:
I added both an uncensored and a censored version just to see (it works with your existing code if you were worried about the ability to plot full uncensored lineages):

![image](https://user-images.githubusercontent.com/20716101/86694972-f566c300-bfc0-11ea-803d-37a2eb6fc210.png)

Reply:
These are the same lineages (same as in terms of parameters used to generate them), the left is uncensored, the right is censored.

Reply:
Lemme know if anything is off to you about this or if you think something is wrong.